internal/pipeline/examples/simplepackage/configs/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha1
```

Comment:
we need to use newer version of Kptfile here v1alpha2

Reply:
done